src/main/scala/temple/generate/docs/internal/DocumentationGeneratorUtils.scala

```diff
@@ -0,0 +1,103 @@
+package temple.generate.docs.internal
+
+import io.circe.Json
+import temple.ast.AbstractAttribute
+import temple.ast.AttributeType._
+import temple.generate.CRUD
+import temple.generate.CRUD.CRUD
+import io.circe.syntax._
+
+import scala.Option.when
+
+object DocumentationGeneratorUtils {
+
+  private[docs] def httpMethod(crud: CRUD): String = crud match {
+    case CRUD.List | CRUD.Read | CRUD.Identify => "GET"
+    case CRUD.Create                           => "POST"
+    case CRUD.Update                           => "PUT"
+    case CRUD.Delete                           => "DELETE"
+  }
+
+  private[docs] def httpCodeString(code: Int): String = code match {
+    case 200 => "200 OK"
+    case 302 => "302 FOUND"
+    case 400 => "400 BAD REQUEST"
+    case 401 => "401 UNAUTHORIZED"
+    case 403 => "403 FORBIDDEN"
+    case 404 => "404 NOT FOUND"
+    case 500 => "500 INTERNAL SERVER ERROR"
+    case v   => throw new RuntimeException(s"Unhandled HTTP Code $v")
```

Comment:
v?

Reply:
v for the first key that jay happened to press on his keyboard 😉